ui/src/app/components/target-details-form/target-details-form.component.scss

```diff
@@ -0,0 +1,9 @@
+.mat-mdc-form-field{
```

Comment:
How is this different from the default css?

Reply:
-If this is not written.

![Screenshot from 2024-06-10 10-20-10](https://github.com/GoogleCloudPlatform/spanner-migration-tool/assets/171904753/10753ea9-2de2-411e-89c1-e6b26e68c715)

-It makes the textbox width to full if I include this.

Reply:
Understood, looks good